src/add/observable/combineLatest.ts

```diff
@@ -1,6 +1,4 @@
 import {Observable} from '../../Observable';
 import {combineLatestStatic} from '../../operator/combineLatest';
 
-Observable.combineLatest = combineLatestStatic;
-
-export var _void: void;
```

Comment:
Refer https://github.com/ReactiveX/RxJS/issues/1010 I mentioned. If there isn't any export makes d.ts to be empty, it makes consumer in trouble. So question's is there anything exported after removing this and type definition's published. I assume operator includes augmentation does.